src/validation.c

```diff
@@ -71,7 +71,7 @@ is_hostname(const char *hostname)
 gboolean
 is_wireguard_key(const char* key)
 {
-    /* Check if this is (most likely) a 265bit, base64 encoded wireguard key */
+    /* Check if this is (most likely) a 256 bit, base64 encoded wireguard key */
```

Comment:
Consider revising the regex pattern in `is_wireguard_key` to ensure it accurately validates all possible valid base64 encoded keys. This might involve adjusting the pattern to match any valid base64 string of the correct length, rather than relying on specific character positions and counts. [important]

Reply:
unrelated to this PR